.gitignore

```diff
@@ -62,3 +65,6 @@ dkms.conf
 /packages
 /out/build/x64-Debug
 /sgKey.snk
+/out/build/x64-Debug
```

Comment:
already add x64-Debug, you do not need to add the files in that folder, all files in that folder will be ignored.